diff -Naru a/arch/arm/dts/sun8i-h3-nanopi-neo.dts b/arch/arm/dts/sun8i-h3-nanopi-neo.dts
--- a/arch/arm/dts/sun8i-h3-nanopi-neo.dts	2020-04-13 23:02:18.000000000 +0800
+++ b/arch/arm/dts/sun8i-h3-nanopi-neo.dts	2021-01-11 21:22:39.721524082 +0800
@@ -43,8 +43,8 @@
 #include "sun8i-h3-nanopi.dtsi"
 
 / {
-	model = "FriendlyARM NanoPi NEO";
-	compatible = "friendlyarm,nanopi-neo", "allwinner,sun8i-h3";
+	model = "GATHER NANO V1.1";
+	compatible = "allwinner,sun8i-h3";
 };
 
 &ehci0 {
diff -Naru a/configs/nanopi_neo_defconfig b/configs/nanopi_neo_defconfig
--- a/configs/nanopi_neo_defconfig	2021-01-11 21:27:17.470423515 +0800
+++ b/configs/nanopi_neo_defconfig	2021-01-11 21:29:45.573302033 +0800
@@ -7,7 +7,5 @@
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_CONSOLE_MUX=y
 CONFIG_DEFAULT_DEVICE_TREE="sun8i-h3-nanopi-neo"
-CONFIG_SUN8I_EMAC=y
-CONFIG_USB_EHCI_HCD=y
-CONFIG_USB_OHCI_HCD=y
 CONFIG_MMC_SUNXI_SLOT_EXTRA=2
+CONFIG_BOOTDELAY=0
