diff --color -Naru a/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts b/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts
--- a/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts	2021-03-03 23:53:51.676896461 +0800
+++ b/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts	2021-06-09 18:04:28.688629247 +0800
@@ -167,3 +167,29 @@
 	usb0_id_det-gpios = <&pio 6 12 GPIO_ACTIVE_HIGH>; /* PG12 */
 	status = "okay";
 };
+
+&spi0 {
+        status = "okay";
+        
+        mcp3201_00: mcp3201@0 {
+			compatible = "microchip,mcp3201";
+			reg = <0>;
+			spi-max-frequency = <1600000>;
+			spi-cpha = <1>;
+			spi-cpol = <1>;
+			#io-channel-cells = <1>;
+		};
+};
+
+&i2c2 {
+        status = "okay";
+
+		ssd1306: oled@3c {
+			compatible = "solomon,ssd1306fb-i2c";
+			reg = <0x3c>;
+			solomon,width = <128>;
+			solomon,height = <64>;
+			solomon,page-offset = <0>;
+			solomon,com-invdir;
+		};
+};
