diff -Naru a/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts b/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts
--- a/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts	2021-11-17 21:04:32.790861382 +0800
+++ b/arch/arm/boot/dts/sun8i-h3-nanopi-neo.dts	2021-11-17 21:07:22.949401926 +0800
@@ -99,6 +99,12 @@
 			gpios = <&pio 6 6 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
+
+		wlan-pwr {
+			label = "wlan-pwr";
+			gpios = <0x09 0x00 0x07 0x00>;
+			default-state = "on";
+		};
 	};
 };
 
